app/facades/websocket/WebSocketController.scala

```diff
@@ -251,6 +299,28 @@ class WebSocketController @Inject() (actorSystem:  ActorSystem,
    */
   private def getDsId(request: RequestHeader) = request.queryString("dsId").head
 
+  /**
+    * Extracts `salt` from the requests query string
+    *
+    * @param request
+    * @return
+    */
+  private def getSalt(request: RequestHeader) = request.queryString("salt").head
+
+  /**
+    * Extracts "auth" parameter from URL
+    *
+    * @param request
+    * @return
+    */
+  private def getAuth(request: RequestHeader): Option[String] = {
```

Comment:
Can be simplified as:
`private def getAuth(request: RequestHeader) = request.queryString("auth").headOption`

Reply:
Thanks, really useful.

Reply:
Thank you, changed.

Reply:
@uralian , request.queryString("auth") throws an exception if "auth" does not exist, so i leave "try" statement.